test/browser_tests/page_objects/page_dbwu-upcoming-procurement-needs.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+function UpcomingProcurementNeeds() {
+
+  this.get = function() {
+    browser.get( 'doing-business-with-us/upcoming-procurement-needs/' );
+  };
+
+  this.pageTitle = browser.getTitle;
+
+  this.sideNav = element( by.css( '.nav-secondary' ) );
+
+  this.mainTitle = element( by.css( '[data-qa-hook="main-title"]' ) );
+
+  this.mainSummary = element( by.css( '[data-qa-hook="main-summary"]' ) );
+
+  this.procurementNeedsExternalLink =
+  element( by.css( '[data-qa-hook="main-summary"] + .short-desc a' ) );
+
+  this.procurementNeedsTable =
+  element( by.css( '.content_main .simple-table' ) );
+
+  this.moreInfoSection =
+  element.all( by.css( '[data-qa-hook="more-info"]' ) );
+
+  this.moreInfoSectionTitles = this.moreInfoSection.all( by.css( 'h1' ) );
```

Comment:
This markup will change. There's only one h1 on the page now and that's the page title. Maybe a data-hook instead?